lib/langchain/processors/csv.rb

```diff
@@ -22,6 +33,29 @@ def parse(data)
       def separator
         @options[:col_sep] || ","
       end
+
+      def chunk_mode
+        if @options[:chunk_mode].to_s.empty?
+          CHUNK_MODE[:row]
+        else
+          raise InvalidChunkMode unless CHUNK_MODE.value?(@options[:chunk_mode])
+
+          @options[:chunk_mode]
+        end
+      end
+
+      def chunk_row(data)
+        ::CSV.new(data.read, col_sep: separator).map do |row|
+          row
+            .compact
+            .map(&:strip)
+            .join(separator)
+        end.join("\n\n")
```

Comment:
From what I can tell, by default if a string is separated by `\n\n` then it gets chunked when it's acted on by `Langchain::Chunker::Text` Let me know if that's incorrect or you prefer a different approach